frontend/app/src/lib/api/generated/data-contracts.ts

```diff
@@ -434,6 +434,18 @@ export interface CreateEventRequest {
   additionalMetadata?: object;
 }
 
+export interface BulkCreateEventRequest {
+  events: CreateEventRequest[];
+}
+
+export interface BulkCreateEventResponse {
+  metadata: APIResourceMeta;
+  /** The events. */
+  events: Event[];
+  /** The pagination information. */
+  pagination: PaginationResponse;
```

Comment:
we're paginating the create response?

Reply:
I wasn't sure about this. If I include that does it magically paginate the response ? For many events maybe it would make sense to paginate (like 1000s) but probably not required.

Reply:
no, we still need the query to page the results but the bigger problem is the ids generated are ephemeral and we dont have a way to GET the next page. I'd just return a list of ids probably

Reply:
Which ids are ephemeral?

Reply:
the ids returned from the copy... in other words we cant return the next page unless we store it somewhere